frontend/tests/acceptance/sysupport-admin-test.js

```diff
@@ -0,0 +1,73 @@
+import { module, test } from 'qunit'
+import { setupApplicationTest } from 'ember-qunit'
+import {
+  authenticateSession,
+  invalidateSession
+} from 'customer-center/tests/helpers/ember-simple-auth'
+import startApp from '../helpers/start-app'
+import destroyApp from '../helpers/destroy-app'
+
+module('Acceptance | Sysupport Admin', function(hooks) {
+  setupApplicationTest(hooks)
+
+  hooks.beforeEach(function() {
+    this.application = null
+  })
+
+  hooks.beforeEach(async function() {
+    this.application = startApp()
+    let user = server.create('user', 'admin')
+    await authenticateSession(this.application, { data: user.id })
+  })
```

Comment:
Use new style tests or refactor to only use one `beforeEach`

Reply:
:+1: